pom.xml

```diff
@@ -114,6 +115,7 @@ LICENSE file.
     <module>asynchbase</module>
     <module>azuretablestorage</module>
     <module>cassandra</module>
+    <module>cloudspanner</module>
```

Comment:
I discussed this internally within Google and the decision was that the official name of the product is "Cloud Spanner" rather than "Google Cloud Spanner" and thus we'd like it to be called just "cloudspanner" here.